products/opensuse/main.pm

```diff
@@ -95,6 +95,9 @@ if (check_var('DESKTOP', 'kde') && !get_var('KDE4')) {
     set_var("PLASMA5", 1);
 }
 
+# SELinux is now default LSM in enforcing mode (bsc#1230118)
+set_var('SELINUX', 1) if is_tumbleweed;
```

Comment:
There will still be apparmor tests - just the other way around (picking intentionally AppArmor instead of selinux) - forcibly setting SELINUX=1 sounds like a recipe to break that

Reply:
true, will check and adjust